test/test_testing.py

```diff
@@ -4619,93 +4693,107 @@ def setUp(self) -> None:
                 test-storage:
                     type: filesystem
             ''')
-        self.harness.begin()
-        self.harness.set_can_connect("test-container", True)
-        self.root = self.harness.get_filesystem_root("test-container")
-        self.container = self.harness.charm.unit.get_container("test-container")
+        harness.begin()
+        harness.set_can_connect("test-container", True)
+        yield harness
+        harness.cleanup()
 
-    def tearDown(self) -> None:
-        self.harness.cleanup()
+    @pytest.fixture
+    def root(self, harness: ops.testing.Harness[ops.CharmBase]):
```

Comment:
The fixture name is a little generic.

Maybe because I'm new, but I can't quite grok what the path refers to when I read the tests that use it. Is it root as in `/` or root of a cgroup or what?

There's an implicit dependency on the `test-container`, which I guess is the local convention, so maybe it's OK?

Reply:
Yeah this is a poor naming. I will change it to something else, maybe `container_fs_root`.